plugin-cerberus/src/main/java/ca/on/oicr/gsi/shesmu/cerberus/BaseCerberusFileProvenanceSkippedRecord.java

```diff
@@ -0,0 +1,149 @@
+package ca.on.oicr.gsi.shesmu.cerberus;
+
+import ca.on.oicr.gsi.cerberus.fileprovenance.ProvenanceRecord;
+import ca.on.oicr.gsi.provenance.model.LimsProvenance;
+import ca.on.oicr.gsi.shesmu.gsicommon.CerberusFileProvenanceSkippedValue;
+import ca.on.oicr.gsi.shesmu.gsicommon.IUSUtils;
+import ca.on.oicr.gsi.shesmu.plugin.AlgebraicValue;
+import ca.on.oicr.gsi.shesmu.plugin.Tuple;
+import ca.on.oicr.gsi.vidarr.api.ProvenanceWorkflowRun;
+import com.fasterxml.jackson.databind.JsonNode;
+import java.nio.file.Path;
+import java.time.Instant;
+import java.util.Map;
+import java.util.Set;
+import java.util.TreeMap;
+import java.util.TreeSet;
+import java.util.stream.Collectors;
+
+abstract class BaseCerberusFileProvenanceSkippedRecord<T extends LimsProvenance>
```

Comment:
Could you add a `skip` field to the SkippedRecord? We had talked about joining cerberus_fp to cerberus_fp_skipped in some use cases, and it would no longer possible to tell if a record is skipped.